### DISCUSSION TITLE
Getting value of ItemInput

### DISCUSSION THREAD

**Discussion by Angenieer** [forntoh/LcdMenu#253] [Q&A] [answered]
Hi! First of all, thank you for this great library!

I can't figure out how to get the value of the ItemInput object. I've tried this:
`    
MENU_SCREEN(pendienteScreen, pendienteItems,
    ITEM_INPUT("Pendiente:", [](char* value){
        //  
    }));
`
in the loop:
1)
` Serial.println(pendienteScreen[0].getValue());
  //Serial.println(pendienteScreen->getItemAt(0)->getValue());
`
Or alternatively:
`pendienteScreen[0].setValue("test");
 //pendienteScreen->getItemAt(0)->setValue("test);
`
and I get the following error:  Compilation error: 'class MenuItem' has no member named 'getValue'.
I've also tried:
2) 
`pendienteItems[0]->getValue();
`
getting the same error:  Compilation error: 'class MenuItem' has no member named 'getValue'.

I can't find neither and answer in other discussions nor an example code in the wiki, in order to solve the problem.
Basically, I'm not understanding how to reference to the object ItemInput. Sorry if it is obvious, i'm a beginner with C++ and Arduino. Thanks!

**Comment by forntoh** [comment, marked as answer]
You will need to do a static cast, if you're certain that the item at index `0` is indeed an `ItemInput`

```c++
static_cast<ItemInput*>(pendienteItems[0])->getValue();
```

Also worth looking at the [SimpleInput example](https://lcdmenu.forntoh.dev/reference/samples/SimpleInput.html)

**Comment by Angenieer** [comment]
It worked! Thank you!
I have another question. Is modifying the value of an ItemInput the best way to print a variable on the LCD? This is what I want:
` pendiente: N`
where N is the value of the variable I want to display on the LCD but I don't want any input from the user. So is there a better way to do this?
